extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java

```diff
@@ -101,12 +101,12 @@
     private static final int SOCKET_TIMEOUT = 5000;
 
     /**
-     * Default notification timeout
+     * Default notification timeout (in seconds)
      */
-    private static final Integer DEFAULT_NOTIFICATION_TIMEOUT = 40000;
-    
+    private static final Integer DEFAULT_NOTIFICATION_TIMEOUT = 20;
```

Comment:
The config-description XML is using a default of 40 seconds, you change it here to 20 seconds.
I don't know what is better 20 or 40.